src/Our.Umbraco.InnerContent/Web/UI/App_Plugins/InnerContent/js/innercontent.js

```diff
@@ -261,26 +264,42 @@ angular.module("umbraco.directives").directive("innerContentOverlay", [
             };
 
             scope.openContentEditorOverlay = function () {
+                setOverlayClasses(scope.config.propertyAlias, scope.currentItem.contentTypeAlias);
                 scope.contentEditorOverlay.title = "Edit " + scope.currentItem.contentTypeName;
                 scope.contentEditorOverlay.dialogData = { item: scope.currentItem };
                 scope.contentEditorOverlay.show = true;
             };
 
             scope.closeContentEditorOverlay = function () {
+                scope.overlayClasses = [];
                 scope.contentEditorOverlay.show = false;
             };
 
             scope.closeAllOverlays = function () {
+                scope.overlayClasses = [];
                 scope.closeContentTypePickerOverlay();
                 scope.closeContentEditorOverlay();
                 scope.config.show = false;
             };
 
+            function setOverlayClasses() {
+                // Ensures that the "create" and "edit" classes don't conflict.
```

Comment:
I'm not sure what this comment means. How would this conflict? Maybe expand so it's more ovious.